libraries/standard/http/include/http_client.h

```diff
@@ -122,9 +126,9 @@ typedef int32_t (* HTTPTransportRecv_t )( HTTPNetworkContext_t * pContext,
  */
 typedef struct HTTPTransportInterface
 {
-    HTTPTransportRecv_t recv;
-    HTTPTransportSend_t send;
-    HTTPNetworkContext_t * pContext;
+    HTTPTransportRecv_t recv;        /**< Transport receive interface */
+    HTTPTransportSend_t send;        /**< Transport interface send interface. */
+    HTTPNetworkContext_t * pContext; /**< User defined transport interface context. */
```

Comment:
Can this just be `void *`? We don't need to specify a type when we are not responsible for interpreting the time

Reply:
Richard does not want any void pointers